TST: Sometimes test_wcsimcat_wcs_transforms_roundtrip fails with ValueError: Polygon made of too few points

Example log: https://github.com/spacetelescope/tweakwcs/actions/runs/23424464777

```
___________________ test_wcsimcat_wcs_transforms_roundtrip ____________________
[gw0] win32 -- Python 3.13.12 D:\a\tweakwcs\tweakwcs\.tox\py313-numpy22-xdist\Scripts\python.EXE

mock_fits_wcs = WCS Keywords

Number of WCS axes: 2
CTYPE : 'RA---TAN' 'DEC--TAN'
CUNIT : 'deg' 'deg'
CRVAL : 82.0 12.0
CRPIX : 512.0 ...169943749476e-06 7.3135370161917055e-06
CD2_1 CD2_2  : -5.877852522924732e-06 6.8199836006249855e-06
NAXIS : 1024  2048

    def test_wcsimcat_wcs_transforms_roundtrip(mock_fits_wcs):
        x = np.arange(5)
        y = np.random.permutation(5)
        imcat = Table([x, y], names=('x', 'y'))
        corr = FITSWCSCorrector(mock_fits_wcs)
>       w = WCSImageCatalog(imcat, corr)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

tweakwcs\tests\test_wcsimage.py:88: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
.tox\py313-numpy22-xdist\Lib\site-packages\astropy\utils\decorators.py:620: in wrapper
    return function(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
tweakwcs\wcsimage.py:102: in __init__
    self.catalog = catalog
    ^^^^^^^^^^^^
tweakwcs\wcsimage.py:244: in catalog
    self.calc_bounding_polygon()
tweakwcs\wcsimage.py:499: in calc_bounding_polygon
    self._calc_cat_convex_hull()
tweakwcs\wcsimage.py:485: in _calc_cat_convex_hull
    self._polygon = SphericalPolygon.from_radec(ra, dec)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
.tox\py313-numpy22-xdist\Lib\site-packages\spherical_geometry\polygon.py:1052: in from_lonlat
    polygon = SingleSphericalPolygon.from_lonlat(lon, lat,
.tox\py313-numpy22-xdist\Lib\site-packages\spherical_geometry\polygon.py:287: in from_lonlat
    return cls(points, center)
           ^^^^^^^^^^^^^^^^^^^
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = SingleSphericalPolygon(array([[0.13626812, 0.96861094, 0.20790347],
       [0.13626705, 0.96861107, 0.20790353],
       [0.13626812, 0.96861094, 0.20790347]]), None)
points = array([[0.13626812, 0.96861094, 0.20790347],
       [0.13626705, 0.96861107, 0.20790353],
       [0.13626812, 0.96861094, 0.20790347]])
inside = None

    def __init__(self, points, inside=None):
        r"""
        Parameters
        ----------
        points : An Nx3 array of (*x*, *y*, *z*) triples in vector space
            These points define the boundary of the polygon.
    
            It may contain zero points, in which it defines the null
            polygon.  It may not contain one, two or three points.
            Four points are needed to define a triangle, since the
            polygon must be closed.
    
        inside : An (*x*, *y*, *z*) triple, optional
            This point must be inside the polygon.  If not provided, an
            interior point will be calculated.  If the polygon is degenerate,
            the inside point will be set to `None`.
        """
        if points is None or len(points) == 0:
            # handle special case of initializing with an empty list of
            # vertices (ticket #1079).
            self._degenerate = True
            self._inside = None
            self._points = np.asanyarray([])
            return
        else:
            self._degenerate = False
    
        if not np.array_equal(points[0], points[-1]):
            points = list(points[:])
            points.append(points[0])
    
        self._points = points = np.asanyarray(points)
    
        if len(points) < 4:
            self._inside = None
>           raise ValueError("Polygon made of too few points")
E           ValueError: Polygon made of too few points

.tox\py313-numpy22-xdist\Lib\site-packages\spherical_geometry\polygon.py:69: ValueError
```

--- a/tweakwcs/tests/test_wcsimage.py
+++ b/tweakwcs/tests/test_wcsimage.py
@@ -82,7 +82,8 @@ def test_wcsimcat_set_none_catalog(mock_fits_wcs, rect_imcat):
 
 def test_wcsimcat_wcs_transforms_roundtrip(mock_fits_wcs):
     x = np.arange(5)
-    y = np.random.permutation(5)
+    y = np.array([2, 1, 4, 0, 3], dtype=float)
+
     imcat = Table([x, y], names=('x', 'y'))
     corr = FITSWCSCorrector(mock_fits_wcs)
     w = WCSImageCatalog(imcat, corr)

